pkg/plugins/utils/version/filter.go

```diff
@@ -17,16 +17,17 @@ const (
 	SEMVERVERSIONKIND string = "semver"
 	// LATESTVERSIONKIND specifies that we are looking for the latest version of an array
 	LATESTVERSIONKIND string = "latest"
+	// REGEXSEMVERVERSIONKIND use regex to extract versions and semantic version to find version
+	REGEXSEMVERVERSIONKIND string = "regex-semver"
```

Comment:
I made this kebab case, this might not follow convention though.

Reply:
To align with other plugin as you can see [here](https://github.com/updatecli/updatecli/blob/caa6cef9fe86cf172abe53a8e2c8484b2d81ed5c/pkg/core/pipeline/resource/main.go#L86), I would use "regex/semver"

Reply:
Perfect, it didn't feel right, and I didn't really engage my brain to resolve it.